components/home/Hero.vue

```diff
@@ -28,15 +39,34 @@
           style="max-width: 98%; border-radius: 12px"
           class="mt-3"
         >
-          <source src="/videos/hero-animation.webm" type="video/webm" />
+          <source :src="video?.url" type="video/webm" />
         </video>
       </div>
     </div>
   </section>
 </template>
 
 <script>
-export default {}
+export default {
+  props: {
+    title: {
+      type: Array,
+      default: () => [],
+    },
```

Comment:
variable name is singular?

Reply:
Its a single title, so the variable name is singular.